package.json

```diff
@@ -10,13 +10,10 @@
   },
   "repository": {
     "type": "git",
-    "url": "https://github.com/vuelidate/vuelidate.git"
+    "url": "https://github.com/yk1711/vuelidate.git"
   },
   "dependencies": {
-    "vue-demi": "^0.12.0"
-  },
-  "optionalDependencies": {
-    "@vue/composition-api": "^1.3.1"
+    "vue-demi": "^0.13.1"
```

Comment:
It's not optional, we need it always.

Reply:
`vue-demi` ? It's not in optional, it just seems that way from the diff :)